bin/run_tests

```diff
@@ -140,6 +140,7 @@ test_signing() {
 test_validator() {
     echo "[---Running validator tests---]"
     run_docker_test ./validator/tests/unit_validator.yaml -s validator -p $ISOLATION_ID
+    run_docker_test shutdown-smoke.yaml -s integration_test -p $ISOLATION_ID
 }
```

Comment:
In that file there are a mix of .yaml and not. It seems since the "compose_file" arg is being passed directly into docker-compose, in run_docker_test, we would be relying on docker-compose to figure out that a particular file has .yaml at the end or .yml. It does do this, but I don't think we should rely upon it.